error.spt

```diff
@@ -19,8 +19,8 @@ if isinstance(response, LazyResponse):
     response.render_body(state)
 err = response.text
 if code == 500 and not err:
-    err = _("Looks like you've found a bug! Sorry for the inconvenience, we'll get it fixed ASAP!")
-
+    err = _("Looks like you've found a bug. It 
```

Comment:
Less shouty, less apologetic, shorter, no self-referencing and confusion, fewer errors in translation from people not carrying over "ASAP" directly or not understanding what it means.